core/src/main/java/com/sequenceiq/cloudbreak/core/bootstrap/service/host/ClusterHostServiceRunner.java

```diff
@@ -472,7 +472,6 @@ private void decoratePillarWithClouderaManagerAutoTls(Cluster cluster, Map<Strin
         Optional<String> licenseOpt = Optional.ofNullable(account.getClouderaManagerLicenseKey());
         if (licenseOpt.isPresent() && isNotEmpty(licenseOpt.get())) {
             String license = licenseOpt.get();
-            LOGGER.debug("Got license key from UMS: {}", license);
```

Comment:
why did you removed?

Reply:
I have no idea I did not see this before

Reply:
Actually it seems like I may have dragged along some stuff from a commit that was removed, I will fix that.

Reply:
Oh I see it's from the branch being changed, might take me a little to fix.